### DISCUSSION TITLE
Energy TAB - mqtt sensor

### DISCUSSION THREAD

**Discussion by mussoa82** [indomus/forum#1508] [HUB: Home Assistant]
Ciao a tutti, 
ho creato sei sensori mqttper la ricezione di alcuni parametri da inverter fotovoltaico, e nel momento in cui inserisco alcuni di questi nel sistema di accumulo, mi restituisce un errore di "statistics_not_defined". 
Il sensore è stato fatto in questo modo :
```
- platform: mqtt
  name: "ZCS-Battery_energycharging"
  unit_of_measurement: "Wh"
  device_class: energy
  state_class: total
  state_topic: "azzurrozcs/zcsenergycharging"
```

Grazie mille a tutti

**Comment by indomus** [comment]
prova con

```
- platform: mqtt
  name: "ZCS-Battery_energycharging"
  unit_of_measurement: "Wh"
  device_class: energy
  state_class: measurement
  state_topic: "azzurrozcs/zcsenergycharging"
```

e poi accertati di averlo messo in recorder

**Reply by Scialla** [reply]
lo state_class è total_increasing, non total
